### PR TITLE
add exact synonym abbreviation for MONDO:0005180 'Parkinson disease'

### PR DESCRIPTION
**Pull request by twhetzel**
closes https://github.com/monarch-initiative/mondo/issues/7742

The user requested the synonym 'PD' for MONDO:0005180 'Parkinson disease'. 

They also mentioned that 'PD' does not need to be on MONDO:0008199 'late-onset Parkinson disease', however the  abbreviation 'PD' is from the [OMIM](https://omim.org/entry/168600) record so I left it as-is. However, I can see some arguments to mark 'PD' as excluded (despite only 12 of 600 excluded synonyms are also exact synonyms). Let me know if you prefer that EXCLUDE be added to 'PD' on MONDO:0008199 'late-onset Parkinson disease'.

### CODE REVIEW

**Review by sabrinatoro — COMMENTED** [review]
@twhetzel 
This looks good. However, based on the comment, LOPD should also be added as an abbreviation for late onset pompe disease. 

Also, please make sure you answer the requester's answer about acronyms.

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -417738,8 +417739,8 @@ synonym: "GSD type 2, late onset" EXACT [Orphanet:420429]
 synonym: "GSD type 2, late-onset" EXACT [Orphanet:420429]
 synonym: "GSD type II, late onset" EXACT [Orphanet:420429]
 synonym: "GSD type II, late-onset" EXACT [Orphanet:420429]
+synonym: "LOPD" EXACT ABBREVIATION [Orphanet:420429]
 synonym: "Pompe disease, late onset" EXACT [Orphanet:420429]
-synonym: "Pompe disease, late-onset" EXACT [Orphanet:420429]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:417742]: I removed the duplicate synonym

Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:417742]: This synonym is not duplicated. There is a dash between "late" and "onset", which is not in the other synonym. I know it looks like a detail, but if people use a dash in their search, they might not find this term. 
Please reinstate this synonym.

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@twhetzel 
There is one minor change request.

**Review by sabrinatoro — COMMENTED** [review]
@twhetzel Approving and merging. 
Please make sure to answer the user's question about how we handle acronyms.

**Review by sabrinatoro — APPROVED** [review]

**Pull request merged**